content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -148,29 +156,27 @@ The preceding command creates a
 
     The response to a successful request is a hello message:
 
-        Hello Kubernetes!
-
-
-
+    ```shell
+    Hello Kubernetes!
+    ```
 
 ## {{% heading "cleanup" %}}
 
-
 To delete the Service, enter this command:
 
+    ```shell
```

Comment:
Hi @RA489 . The updates look great! Can you remove the indentation for the last two code blocks?
See, https://deploy-preview-25080--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/expose-external-ip-address/#cleaning-up

Reply:
sure

Reply:
No indentation for  the above code block is needed.
Indentation by 4 spaces is an old practice for code hilighting.

Reply:
@kbhawkey removed the indentation for the last two code blocks.